packages/web/src/pages/chat-page/components/ChatComposer.tsx

```diff
@@ -86,11 +90,12 @@ export const ChatComposer = (props: ChatComposerProps) => {
 
   // Set focus and clear on new chat selected
   useEffect(() => {
-    if (chatId) {
+    if (chatId !== chatIdRef.current) {
```

Comment:
just confirming you've tested this case, seems like we might be resetting `chatIdRef` to current `chatId` on every load so this would always be false? Assuming you want it to be true when we navigate here from the "share via DM" flow.

Reply:
Want it to be false from initial navigation, but true when you click on the sidebar to change the chat. That way this prevents this effect from clearing the default preset message on first render